AddressBook.ts

```diff
@@ -20,6 +20,20 @@ export class AddressBook {
         return Object.values(this.contacts);
     }
 
+    /**
+     * Get blacklisted contacts by a given address
+     */
+    getBlackListedContacts() {
+        return Object.values(this.contacts).filter(contact=>contact.isBlackListed);
+    }
+
+    /**
+     * Get whitelisted contacts by a given address
+     */
+    getWhiteListedContacts() {
+        return Object.values(this.contacts).filter(contact=>!contact.isBlackListed);
+    }
```

Comment:
should you add any unit tests for these new functions?

Reply:
Done.